src/controllers/admin.controller.js

```diff
@@ -1,53 +1,71 @@
+const adminCtrl = {};
+
 // MICROSERVICE - HEROKU - UNIVERSITY
 const universityServiceAPI = require("../services/at-university-api.service");
 
-const adminCtrl = {};
-
+// AT-UNIVERSITY - Admin - Index
 adminCtrl.renderIndexAdmin = (req, res) => {
-    res.render('admin/dashboard');
-}
+    res.render("admin/index");
+};
+
+// AT-UNIVERSITY - Admin - Render Course List
+adminCtrl.renderCourseList = async(req, res) => {
+    try {
+        const responseCourses = await universityServiceAPI.getAllCourses();
+        console.log("---> adminCtrl.renderCourseList.getAllCourses");
+        //console.log(responseCourses.data);
+        const courses = responseCourses.data;
+        res.render("admin/course/index", { courses });
+    } catch (err) {
+        console.error(err.message);

```

Comment:
Is this error handling necessary?
I reviewed the implementation of `getAllCourses` and you are handling errors there, please double check your implementation.

Reply:
oh yes, imagine if universityServiceAPI.getAllCourses(); return a timeout

Reply:
Great, thank you for double check 👌